services/github/github-forks.service.js

```diff
@@ -47,44 +50,28 @@ module.exports = class GithubForks extends LegacyService {
 
   static get defaultBadgeData() {
     return {
+      label: 'forks',
       namedLogo: 'github',
     }
   }
 
-  static registerLegacyRouteHandler({ camp, cache, githubApiProvider }) {
-    camp.route(
-      /^\/github\/forks\/([^/]+)\/([^/]+)\.(svg|png|gif|jpg|json)$/,
-      cache((data, match, sendBadge, request) => {
-        const user = match[1] // eg, qubyte/rubidium
-        const repo = match[2]
-        const format = match[3]
-        const apiUrl = `/repos/${user}/${repo}`
-        const badgeData = getBadgeData('forks', data)
-        if (badgeData.template === 'social') {
-          badgeData.logo = getLogo('github', data)
-          badgeData.links = [
-            `https://github.com/${user}/${repo}/fork`,
-            `https://github.com/${user}/${repo}/network`,
-          ]
-        }
-        githubApiProvider.request(request, apiUrl, {}, (err, res, buffer) => {
-          if (githubCheckErrorResponse(badgeData, err, res)) {
-            sendBadge(format, badgeData)
-            return
-          }
-          try {
-            const data = JSON.parse(buffer)
-            const forks = data.forks_count
-            badgeData.text[1] = forks
-            badgeData.colorscheme = undefined
-            badgeData.colorB = '#4183C4'
-            sendBadge(format, badgeData)
-          } catch (e) {
-            badgeData.text[1] = 'invalid'
-            sendBadge(format, badgeData)
-          }
-        })
-      })
-    )
+  static render({ user, repo, forkCount }) {
+    return {
+      message: metric(forkCount),
+      link: [
+        `https://github.com/${user}/${repo}/fork`,
+        `https://github.com/${user}/${repo}/network`,
+      ],
+    }
+  }
+
+  async handle({ user, repo }) {
+    const { forks_count: forkCount } = await this._requestJson({
+      url: `/repos/${user}/${repo}`,
+      schema,
+      errorMessages: errorMessagesFor(),
+    })
+    console.log('forkCount', forkCount)
```

Comment:
Was this intentionally left in, and if so what do we need it for?

Reply:
Whoops

Reply:
It'd be great to redirect our console.log statements through a logger and/or eslint-ignore them, so the linter can catch stuff like this.